definitions/ext-vpc_network/golden_metrics.yml

```diff
@@ -46,7 +46,7 @@ ingressBytes:
       from: KFlow
       where: "provider = 'kentik-vpc' AND flow_direction = 'ingress'"
     aws-kinesis-firehose/vpc-flow-logs:
-      select: sum(bytes*sample_rate)
-      from: Log
+      select: sum(bytes*numeric(sample_rate))
```

Comment:
same comment re: sample rate and bytes

Reply:
Choosing not to address this one: it's the same logic as the `kentik/aws-vpc-flow-events` metric, so we can revisit both together in the future if it's necessary